shared/CMakeLists.txt

```diff
@@ -25,17 +25,16 @@ set (srcs
 
     String.F90
   )
-esma_add_library (${this} SRCS ${srcs} DEPENDENCIES gftl-shared pflogger MPI::MPI_Fortran)
-#target_include_directories (${this} PRIVATE ${MAPL_SOURCE_DIR}/include)
+esma_add_library (${this} SRCS ${srcs} DEPENDENCIES gftl-shared MPI::MPI_Fortran pflogger)
```

Comment:
What is the significance of changing the order of libraries here?       I generally put the lowest dependencies last, though I'll admit the original version did not even accomplish that.    Was this change necessary or just stylistic?

Reply:
Changing the order was not necessary -- just stylistic. Happy to revert to the previous order.

Reply:
No - at this point, do not revert.   Just important for us to understand if there was a real issue here.

I should probably develop a policy for ordering the items in the list, but the general case probably cannot be uniquely specified in any meaningful manner.    But please do try to refrain from changes that are not needed - except possibly to improve style consistency within the file.